examples/src/bin/ges.rs

```diff
@@ -0,0 +1,94 @@
+extern crate gstreamer as gst;
+use gst::prelude::*;
+
+extern crate gstreamer_editing_services as ges;
+use ges::prelude::*;
+use ges::PipelineExt;
+
+use std::env;
+
+#[allow(unused_imports)]
+#[path = "../examples-common.rs"]
+mod examples_common;
+
+extern crate failure;
+#[allow(unused_imports)]
+use failure::Error;
+
+fn main_loop(uri: &str) -> Result<(), Error> {
+    gst::init()?;
+    ges::init().unwrap();
```

Comment:
can't you get rid of these unwraps?

Reply:
You could simply add another ? here instead :)

Reply:
Ended up getting read of then :-)